fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -861,12 +862,13 @@ public static FHIRSearchContext parseQueryParameters(Class<?> resourceType,
         return parameterValues;
     }
 
-    private static List<QueryParameterValue> parseQueryParameterValuesString(Type type,
-            String queryParameterValuesString) throws FHIRSearchException {
+    private static List<QueryParameterValue> parseQueryParameterValuesString(SearchParameter searchParameter, Type type,
+        String queryParameterValuesString, String modifierResourceTypeName) throws FHIRSearchException {
```

Comment:
the extra indent here is used to distinguish this as a continuation of the method signature and not the first line of the method
```suggestion
            String queryParameterValuesString, String modifierResourceTypeName) throws FHIRSearchException {
```

Reply:
should be fixed